subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -894,9 +894,10 @@ static void nct_mqtt_evt_handler(struct mqtt_client *const mqtt_client,
 	case MQTT_EVT_PUBLISH: {
 		const struct mqtt_publish_param *p = &_mqtt_evt->param.publish;
 
-		LOG_DBG("MQTT_EVT_PUBLISH: id = %d len = %d",
+		LOG_DBG("MQTT_EVT_PUBLISH: id = %d len = %d, topic = %s",
 			p->message_id,
-			p->message.payload.len);
+			p->message.payload.len,
+			p->message.topic.topic.utf8);
```

Comment:
This will be nice to have, though I'm not sure whether name is guaranteed to be null-terminated. You may want to use `%.*s` with `message.topic.topic.size`. Or do you know for sure it is null-terminated?